src/gc/gc.cpp

```diff
@@ -26724,55 +26724,11 @@ BOOL gc_heap::prepare_bgc_thread(gc_heap* gh)
 
 BOOL gc_heap::create_bgc_thread(gc_heap* gh)
 {
-    BOOL ret = FALSE;
-
     assert (background_gc_done_event.IsValid());
 
     //dprintf (2, ("Creating BGC thread"));
 
```

Comment:
BTW: I happened to take closer look at `bgc_thread_stub` and `bgc_thread_function`. There is more stuff would be good candidate for folding into GCToEEInterface into CreateBackgroundThread:

- `GCToEEInterface::SetGCSpecial(heap->bgc_thread);` - this is the only use of this method in GC
- The code under `#ifndef FEATURE_REDHAWK` in `bgc_thread_function`

(It does not need to be fixed in this change.)